migrations/migrations/20190626181121-users.js

```diff
@@ -29,6 +29,11 @@ module.exports = {
       },
       updated_at: {
         type: Sequelize.DATE
+      },
+      role: {
```

Comment:
When you add a new field or changes something of model you create a new migration